main.rb

```diff
@@ -1,39 +1,47 @@
-require_relative 'classroom'
-require_relative 'student'
-require_relative 'book'
-require_relative 'rental'
+# rubocop:disable Metrics/MethodLength
+# rubocop:disable Metrics/CyclomaticComplexity
```

Comment:
- [ ] Kindly remove this and use `rubocop.yml` to configure your rules in order to make your file clean, it's not so professional when we disable linters. More so, commented-out code makes our work look messy. ℹ️